ansible/roles/builder/tasks/main.yml

```diff
@@ -12,6 +12,48 @@
   include_role:
     name: windows-ntp
 
+- name: Install Windows-Containers feature
+  win_feature:
+    name: Containers
+    state: present
+
+- name: Install feature NET-Framework-Features
+  win_feature:
+    name: 'NET-Framework-Features'
+    state: present
+
+- name: Install Hyper-V feature
+  win_feature:
+    name: Hyper-V
+    include_management_tools: true
+    state: present
+  register: hyperv_installation
+
+- name: Reboot the system
+  win_reboot:
+    test_command: powershell.exe 'Get-WindowsFeature'
+    pre_reboot_delay: 15
+    post_reboot_delay: 15
+  when: hyperv_installation.reboot_required
```

Comment:
I think it should be `hyperv_installation.feature_result.reboot_required`. What do you think?